docs/development/code_guide.rst

```diff
@@ -5,14 +5,185 @@ Code Development Guidelines
 This document describes the coding requirements and guidelines to be
 followed during the development of PlasmaPy and affiliated packages.
 
-Dependencies
-============
+Code written for PlasmaPy must be compatible with Python 3.6 and
+later. Python 2 is not supported by PlasmaPy.
+
+PlasmaPy requires 
+
+* Python 3.6 or later
+* Astropy 2.0 or later
+* NumPy 1.13 or later
+* scipy 0.19 or later
+* matplotlib 2.0 or later
+
+Obtaining PlasmaPy source code
+=========================================
+
+After creating your GitHub account, go to the [main
+repository](https://github.com/PlasmaPy/PlasmaPy) and **fork a copy of
+PlasmaPy to your account**.
+
+To access Git commands on Windows, try `Git Bash <https://git-scm.com/downloads>`_.
+
+Next you must **clone your fork to your computer**.  Go to the
+directory that will host your PlasmaPy directory, and run one of the
+following commands (after changing *your-username* to your username).
+If you would like to use HTTPS (which is the default and easier to set
+up), then run:
+
+.. code-block:: bash
+
+  git clone https://github.com/your-username/PlasmaPy.git
+
+SSH is a more secure option, but requires you to [set up an SSH
+key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/)
+beforehand.  The equivalent SSH command is:
+
+.. code-block:: bash
+
+    git clone git@github.com:your-username/PlasmaPy.git
+    cd PlasmaPy
+
+After cloning, we must tell git where the development version of
+PlasmaPy is by running:
+
+.. code-block:: bash
+
+  git remote add upstream git://github.com/PlasmaPy/PlasmaPy.git
+
+To check on which remotes exist, run `git remote -v`.  You should get
+something like this:
+
+.. code-block:: bash
+
+  origin		git@github.com:namurphy/PlasmaPy.git (fetch)
+  origin		git@github.com:namurphy/PlasmaPy.git (push)
+  upstream	git@github.com:PlasmaPy/PlasmaPy.git (fetch)
+  upstream	git@github.com:PlasmaPy/PlasmaPy.git (push)
+
+Setting up an environment for development
+=========================================
+
+Setup procedures for the two most popular virtual
+environments, conda and virtualenv, are listed below.
+
+Conda
+-----
+
+To set up a development environment for PlasmaPy, `the Anaconda
+distribution <https://www.anaconda.com/download/>`_ is strongly
+recommended.
+
+After installing Anaconda, launch any conda environment and enter
+PlasmaPy's repository root directory.
+
+On Windows, the way to do this is via running `Anaconda Prompt` from the
+Start Menu. `Git Bash` may also work if you have added Anaconda to `PATH`.
+
+Afterwards, execute the following:
+
+.. code-block:: bash
+
+    conda env create -f environment.yml
+
+The `environment.yml` file refers to the one located in PlasmaPy's root.
+You may now enter the environment via
+
+.. code-block:: bash
+
+    source activate plasmapy-dev
```

Comment:
This part needs to be changed to be specific to conda.

Reply:
I took a slightly different approach as going first into any conda env, creating the environment via the .yml file and then activating `plasmapy-dev` should let you skip all the library linking... I think. I'm not sure how that part even worked given you weren't in a conda env.

Oh yeah, if you'd like to test this again you may wanna run `conda env remove -n plasmapy-dev` from conda's root to prepare. Sorry for the trouble.